lib/rubocop/cop/flexport/global_model_access_from_engine.rb

```diff
@@ -127,7 +127,7 @@ def in_enforced_engine_file?
 
         def in_disabled_engine?(file_path)
           disabled_engines.any? do |e|
-            file_path.include?(File.join(engines_path, e))
+            file_path.include?(File.join(engines_path, e, ""))
```

Comment:
the effect of this adding a trailing `/` was not obvious to me. maybe a comment to explain why this is needed?